distribution/build.sh

```diff
@@ -5,19 +5,35 @@ set -o errexit
 declare -r here="$(dirname "${BASH_SOURCE[0]}")"
```

Comment:
Can we change the "here" var with something like currentDir?

Reply:
Fixed. I've also put a descriptive comment explaining the use of helm to generate the realm file in `distribution/build.sh`.